<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaEjecucionAlgoritmos</class>
 <widget class="QDialog" name="VentanaEjecucionAlgoritmos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Algoritmos a ejecutar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="layout_algoritmos"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Criterios de parada</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="layout_cp">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="layout_iteraciones">
       <item>
        <widget class="QCheckBox" name="check_tiempo">
         <property name="text">
          <string>Tiempo (s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="spin_tiempo">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>3600.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layout_evaluaciones">
       <item>
        <widget class="QCheckBox" name="check_evaluaciones">
         <property name="text">
          <string>Evaluaciones</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_evaluaciones">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layout_tiempo">
       <item>
        <widget class="QCheckBox" name="check_iteraciones">
         <property name="text">
          <string>Iteraciones</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_iteraciones">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="boton_ejecutar">
     <property name="text">
      <string>Ejecutar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
